src/linpde_gp/linfuncops/diffops/_partial_derivative.py

```diff
@@ -0,0 +1,182 @@
+from collections.abc import Callable
+import functools
+from typing import Union
+
+import jax
+import jax.numpy as jnp
+import probnum as pn
+from probnum.typing import ShapeLike
+
+import linpde_gp  # pylint: disable=unused-import # for type hints
+from linpde_gp.functions import JaxFunction, JaxLambdaFunction
+
+from .._arithmetic import SumLinearFunctionOperator
+from ._coefficients import MultiIndex, PartialDerivativeCoefficients
+from ._lindiffop import LinearDifferentialOperator
+
+
+class PartialDerivative(LinearDifferentialOperator):
+    def __init__(
+        self,
+        multi_index: MultiIndex,
+        *,
+        use_jax_fallback: bool = True,
```

Comment:
We want this to be `True` (imo) when we construct a `PartialDerivative` directly. Any `LinearDifferentialOperator` should have some kind of jax fallback, and so `PartialDerivative` also needs one.
We want it to be `False` when `LinearDifferentialOperator.to_sum` constructs `PartialDerivative` instances. Here, we want `PartialDerivative` to raise an error if its call is not implemented, so that `LinearDifferentialOperator` can then call its own, potentially more efficient jax fallback.